pkg/record/recorder.go

```diff
@@ -51,24 +51,28 @@ type recorder struct {
 	record.EventRecorder
 }
 
+func (r recorder) String(reason string) string {
+	return cases.Title(language.English, cases.NoLower).String(reason)
```

Comment:
```suggestion
	return cases.Title(language.Und).String(reason)
```
nit, we used that in core CAPI (just in case you want to use the same)

xref: https://en.wiktionary.org/wiki/und#:~:text=(international%20standards)%20ISO%20639%2D,code%20for%20an%20undetermined%20language.

Reply:
FYI, switching from `strings.Title` to `cases.Title` without `cases.NoLower` changes the resulting string. We might want to keep `cases.NoLower` here.

Context: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/1608#discussion_r960609376

Reply:
Was the reason why I introduced the lower case option when working on the PR.

Reply:
Thx! I'll test what Und does and then probably follow-up if the same change in core CAPI